spec/support/shared_examples/recurring_contributions/subscriptions_processor.rb

```diff
@@ -54,6 +53,14 @@
     it "should return a subscription instance with errors" do
       expect(subject.errors.any?).to be true
     end
+
+    context "when the donator_cpf is blank" do
+      let(:subscription) { build(:subscription, :bank_billet_payment, donator_cpf: '') }
```

Comment:
I think that `nil` is a better context for presence validation, or you can have both to have a more complete spec